backend/graphql/resolvers/testSessionResolvers.ts

```diff
@@ -41,6 +41,10 @@ const testSessionResolvers = {
       _parent: undefined,
       { teacherId }: { teacherId: string },
     ) => testSessionService.getTestSessionsByTeacherId(teacherId),
+    getTop5StudentsByTestSessionId: (
```

Comment:
```suggestion
    getTopFiveStudentsById: (
```
nit: rename function